ui/src/pages/Apps/styles.scss

```diff
@@ -1,5 +1,17 @@
+.contribute-notes {
+    a:link,
+    a:visited,
+    a:active {
+        color: #002752;
+    }
+
+    a:hover {
+        color: #cc2799;
+    }

```

Comment:
It might be good to have those as site-wide styles (maybe applying them to everything inside the `landing-content` div. E.g. on the homepage there are also unstyled links which could use this too.

Reply:
I could do that. What do you think @daveajones?

Reply:
That would work for me.  I'll go ahead and merge this.